Package.swift

```diff
@@ -8,8 +8,16 @@ let package = Package(
     ],
     targets: [
         .target(
-        	name: "Leanplum", 
-        	path: "Leanplum-SDK/Classes",
-        	publicHeadersPath: ".")
+            name: "Leanplum",
+            path: "Leanplum-SDK",
+            exclude: [
+                "LeanplumSDK_iOS-Info.plist",
+                "LeanplumSDK_tvOS-Info.plist"
+            ],
+            resources: [
+                .process("Resources")
+            ],
+            publicHeadersPath: "Classes/include"
```

Comment:
```suggestion
            ]
```
Since `include` in in the top level directory, this line is not required (and is actually incorrect right now)

Reply:
Good catch, will update the PR